pkg/operation/botanist/secrets.go

```diff
@@ -799,6 +809,30 @@ func (b *Botanist) generateSecrets() ([]interface{}, error) {
 			Bits: 4096,
 		},
 
+		// Secret definition for etcd server
```

Comment:
can we generate a CA for `etcd`? Sharing the `kubernetes` CA is a bad practice.

Reply:
Maybe we can postpone that until the way how we generate certificates/secrets has been refactored? It's already quite messy and the CA part is not well encapsulated into functions.